articles/060_ros_middleware_interface.md

```diff
@@ -170,44 +171,19 @@ However the performance using the DynamicData API will likely always be lower co
 
 When ROS supports different middleware implementations it should be as easy and low effort as possible for users to switch between them.
 
-One obvious way will be for a user to rebuild all ROS packages from source selecting a different middleware implementation.
+#### Decide at compile time
+
+One obvious way will be for a user to build all ROS packages from source selecting a specific middleware implementation.
 While the workflow won't be too difficult (probably half a dozen command-line invocations), it still requires quite some build time.
 
-To avoid the overhead of rebuilding everything a different set of binary packages could be provided.
+To avoid the need to build from source a set of binary packages could be provided which chose the middleware implementation at build time.
```

Comment:
are provided ? looks like that's what we ship in our binaries already

Reply:
While that might be the case right now we are aiming to provide binaries for multiple RMW implementations in the future. Therefore I would claim (and promise) here that we do.

Reply:
good point, let's leave it as is then